dev/integration_tests/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -461,9 +464,13 @@
 			baseConfigurationReference = 9740EEB21CF90195004384FC /* Debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = S8QB4VV633;
```

Comment:
I'd just revert this file.  You don't want to check this in and the other stuff is just formatting.

Reply:
Done